client/src/pages/Receipt.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import Cart from "./../components/Cart";
+import { CartItem } from "./../interfaces";
+
+function Receipt() {
+  // Update URL params to find storeID
+  const curUrl = window.location.search;
```

Comment:
Move to react router dom: https://reacttraining.com/react-router/web/guides/quick-start

Reply:
#57